src/ui.rs

```diff
@@ -681,6 +635,90 @@ impl<'a> UiFrame<'a> {
     }
 }
 
+fn draw_var_combo_box(
```

Comment:
Does Rust give you hell if you move this into `UiFrame`?

Reply:
I didn't even try honestly. It should work afaik, it's just that for some reason I default to free functions instead of associated functions when the function is going to be private anyway. I can make it an associated function if you want, no strong feelings.